libafl/src/executors/hooks/inprocess.rs

```diff
@@ -247,15 +233,19 @@ impl<I, S> InProcessHooks<I, S> {
         unsafe {
             setup_signal_handler(data)?;
         }
+
+        #[cfg(feature = "std")]
+        unsafe {
+            let data = &raw mut GLOBAL_STATE;
+            (*data).crash_handler =
```

Comment:
This guy would still be racey, right?

`new` shoudl probably be `unsafe`?

Reply:
Or are the crash handler always exactly the same now, anyway?

Reply:
always the same. it never change